CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/editors/AbstractEditorFactory.java

```diff
@@ -67,6 +68,7 @@ public abstract static class AbstractEditor<V> {
         protected V currentValue;
         protected Node editorHeading = null;
         protected Node editorControls = null;
+        protected CheckBox noValueCheckBox = null;
```

Comment:
given the new changes, are you able to move this back into the editor factories they were removed from (given not all editor factories require the `noValueCheckBox` , it probably shouldn't stay in the the abstract one)